tests/integration/test_ask_tell_optimization.py

```diff
@@ -47,26 +49,58 @@
 # We use a copy of these for a quicker test against a simple quadratic function
 # (copying is necessary as some of the acquisition rules are stateful).
 OPTIMIZER_PARAMS = (
-    "num_steps, reload_state, acquisition_rule_fn",
+    "num_steps, reload_state, acquisition_rule_fn, num_models",
     [
         pytest.param(
-            20, False, lambda: EfficientGlobalOptimization(), id="EfficientGlobalOptimization"
+            20, False, lambda: EfficientGlobalOptimization(), 1, id="EfficientGlobalOptimization"
         ),
         pytest.param(
             20,
             True,
             lambda: EfficientGlobalOptimization(),
+            1,
```

Comment:
It's annoying that we have to put these everywhere for the non-local tests. Is there any way to infer it from the acquisition rule?

Reply:
I don't think there is a neat way to infer this from the rule. For example, the tests 'BatchTrustRegionBox' and 'BatchTrustRegionBox/LocalModels' create almost identical rules, except one is using local models. We could choose an artificial difference (e.g. num_query_points) but that doesn't seem good. I tried to find out if there is a way to have default values with `pytest.parametrize``, but couldn't find a way. Do you have any other ideas?

Reply:
I agree BTW that it is annoying.